src/lib.rs

```diff
@@ -110,37 +143,52 @@ impl Build {
             fs::remove_dir_all(&build_dir_path)?;
         }
 
-        log::trace!("Build: Creating new build directory");
-        fs::create_dir(&build_dir_path).context("cannot create build directory")?;
+        log::trace!("Creating new build directory");
+        fs::create_dir_all(&build_dir_path).context("cannot create build directory")?;
 
-        log::trace!("Build: Writing files into build directory");
+        log::trace!("Writing files into build directory");
         fs::write(wasm_js_path, wasm_js).with_context(|| "cannot write js file")?;
         fs::write(wasm_bin_path, wasm_bin).with_context(|| "cannot write WASM file")?;
 
         let mut copy_options = fs_extra::dir::CopyOptions::new();
         copy_options.overwrite = true;
         copy_options.content_only = true;
 
-        log::trace!("Build: Copying static directory into build directory");
-        fs_extra::dir::copy(static_dir_path, build_dir_path, &copy_options)
-            .context("cannot copy static directory")?;
+        if let Some(static_dir) = self.static_dir_path {
+            log::trace!("Copying static directory into build directory");
+            fs_extra::dir::copy(static_dir, &build_dir_path, &copy_options)
+                .context("cannot copy static directory")?;
+        }
 
-        Ok(())
+        log::info!("Builded successfully at {}", build_dir_path.display());
+
+        Ok(build_dir_path)
     }
 }
 
-#[derive(Debug, Clone, StructOpt)]
+#[non_exhaustive]
+#[derive(Debug, StructOpt)]
 pub struct Watch {
     #[structopt(long = "watch", short = "w")]
-    watch_paths: Vec<PathBuf>,
+    pub watch_paths: Vec<PathBuf>,
```

Comment:
Here it's nice to have them public because it allows the used to delete elements in the vector